internals/overlord/cmdstate/request.go

```diff
@@ -70,8 +70,14 @@ func Exec(st *state.State, args *ExecArgs) (*state.Task, ExecMetadata, error) {
 		return nil, ExecMetadata{}, errors.New("cannot use interactive mode without a terminal")
 	}
 
-	// Inherit the pebble daemon environment.
+	// Inherit the pebble daemon environment (execpt HOME and USER env vars)
```

Comment:
```suggestion
	// Inherit the pebble daemon environment (except HOME and USER env vars)
```

Reply:
addressed in [df9e315](https://github.com/canonical/pebble/pull/262/commits/df9e315f921d412ecc9ccd9718279d0dc14e2be8)